### ISSUE TITLE
Review: 2.4.7 Focus Visible (AA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#55]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.4.7 considering the mobile research questions (to be completed)

---
Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.
---
Sufficient Techniques for Success Criterion 2.4.7

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    G149: Using user interface components that are highlighted by the user agent when they receive focus
    C15: Using CSS to change the presentation of a user interface component when it receives focus
    G165: Using the default focus indicator for the platform so that high visibility default focus indicators will carry over
    G195: Using an author-supplied, visible focus indicator
    C40: Creating a two-color focus indicator to ensure sufficient contrast with all components
    [SCR31: Using script to change the background color or border of the element with focus ](https://www.w3.org/WAI/WCAG22/Techniques/client-side-script/SCR31.html)

Failures for Success Criterion 2.4.7

    F55: Failure of Success Criteria 2.1.1, 2.4.7, and 3.2.1 due to using script to remove focus when focus is received
    F78: Failure of Success Criterion 2.4.7 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator
